core/chain.go

```diff
@@ -12,49 +13,9 @@ import (
 	chantypes "github.com/cosmos/ibc-go/v8/modules/core/04-channel/types"
 	ibcexported "github.com/cosmos/ibc-go/v8/modules/core/exported"
 	"github.com/hyperledger-labs/yui-relayer/log"
+	"go.opentelemetry.io/otel/trace"
 )
 
-// ProvableChain represents a chain that is supported by the relayer
```

Comment:
I have moved the code related to `ProvableChain` to core/provable-chain.go and added some methods for tracing.

Reply:
> added some methods for tracing.

I have removed these methods and added tracing bridges in https://github.com/hyperledger-labs/yui-relayer/pull/162/commits/f048ca52b41f08c778b166d3f91e2b2d447627b7